install/kubernetes/helm/meshery/templates/ingress.yaml

```diff
@@ -29,16 +35,25 @@ spec:
   {{- end }}
 {{- end }}
   rules:
-  {{- range .Values.ingress.hosts }}
-    - host: {{ .host | quote }}
+    {{- range .Values.ingress.hosts }}
+    - host: {{ tpl .host $ | quote }}
       http:
         paths:
-        {{- range .paths }}
-          - path: {{ . }}
-            pathType: {{ $pathType }}
+          {{- range .paths }}
+          - path: {{ .path }}
+            {{- if and .pathType (eq $apiVersion "networking.k8s.io/v1") }}
```

Comment:
Is this comparing pathType and apiVersion?

Reply:
If so, pathType will either be "ImplementationSpecific" or "Prefix"

Reply:
No, its checking if the `apiVersion` is "networking.k8s.io/v1" AND `.pathType` is specified.

Reply:
I have already added a default pathType as Prefix in values.yaml 👍